schema/config_pointset.json

```diff
@@ -1,15 +1,17 @@
 {
   "title": "Pointset Config",
+  "description": "[Read more](../../docs/messages/pointset.md#config)",
```

Comment:
Generally speaking the "read more" is redundant because that's what a link is, so ideally would be descriptive of what it is, e.g. "[Pointset Config Documentation]" -- which then indicates what the link is going to.

Reply:
I've changed all instances where the link is the only description. I've retained those where they're at the end of the a sentence, e.g. `The `state_etag` of the last _state_ message sent by the device. [Read more](../../docs/specs/sequences/writeback.md)`.

Reply:
Actually I've updated it so it includes the descriptive of what it is in all instances, e.g. [Read more - writeback documentation] for when it's at the end of a sentence